src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -7618,25 +7618,23 @@ private struct REPARSE_GUID_DATA_BUFFER
         /// <returns>The target of the reparse point</returns>
         public static IEnumerable<string> GetTarget(PSObject instance)
         {
-            FileSystemInfo fileSysInfo = instance.BaseObject as FileSystemInfo;
-
-            if (fileSysInfo != null)
+            if (instance.BaseObject is FileSystemInfo fileSysInfo)
             {
-                if (Platform.IsWindows)
+#if !UNIX
+                using (SafeFileHandle handle = OpenReparsePoint(fileSysInfo.FullName, FileDesiredAccess.GenericRead))
                 {
-                    using (SafeFileHandle handle = OpenReparsePoint(fileSysInfo.FullName, FileDesiredAccess.GenericRead))
-                    {
-                        string linkTarget = InternalGetTarget(handle);
+                    string linkTarget = WinInternalGetTarget(handle);
 
-                        if (linkTarget != null)
-                            return (new string[] { linkTarget });
+                    if (linkTarget != null)
+                    {
+                        return (new string[] { linkTarget });
                     }
                 }
-
+#endif
                 return InternalGetTarget(fileSysInfo.FullName);
```

Comment:
I see the definition for this method is gone but a call remains. Is there another definition somewhere?

Reply:
Yes, one - wih handle parameter, second with string parameter. First one was removed.

Reply:
Just checking :smile: